<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskDialog</class>
 <widget class="QDialog" name="TaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>任务编辑</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxBasic">
     <property name="title">
      <string>基本信息</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>标题:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditTitle">
        <property name="placeholderText">
         <string>请输入任务标题</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCategory">
        <property name="text">
         <string>分类:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxCategory">
        <property name="placeholderText">
         <string>请选择分类</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPriority">
        <property name="text">
         <string>优先级:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="widgetPriority" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelStatus">
        <property name="text">
         <string>状态:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QWidget" name="widgetStatus" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTime">
     <property name="title">
      <string>时间安排</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelStartTime">
        <property name="text">
         <string>开始时间:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEditStart">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelDeadline">
        <property name="text">
         <string>截止时间:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEditDeadline">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelRemind">
        <property name="text">
         <string>提醒时间:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEditRemind">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTags">
     <property name="title">
      <string>标签管理</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widgetTags" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEditNewTag">
          <property name="placeholderText">
           <string>输入新标签，用逗号分隔</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAddTag">
          <property name="text">
           <string>添加标签</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxDescription">
     <property name="title">
      <string>详细描述</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTextEdit" name="textEditDescription">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>请输入任务详细描述...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
